scripts/README.md

```diff
@@ -0,0 +1,3 @@
+# Scripts
+
+**#TODO: Explain why we have this directory and what's in it**
```

Comment:
This is a directory with scripts for use with utils/controller_debug.py. I think ideally we should consolidate it into a top-level directory debug/{controller_debug.py, scripts/{...}}

Reply:
I agree, it would probably be best to move the controller_debug.py into it's own directory which would allow it to be more easily split into multiple Python files which would help.  the scripts directory could be a sub-directory under that, and it would be nice to have a second scripts directory (local_scripts maybe) where scripts that don't really belong in the repo should live.  That local one could be in .gitignore.

Reply:
Created a ticket about documenting controller_debug.py.